ui/src/app/edge/live/live.component.html

```diff
@@ -50,6 +50,10 @@
               <fixdigitaloutput [componentId]="widget.componentId"></fixdigitaloutput>
             </ion-col>
 
+            <ion-col *ngSwitchCase="'Controller.CHP.SoC'" size="6">
```

Comment:
This should also be `Controller.Chp.Soc` to be more in line with all the other Controllers. Will require refactoring even in Java side, but might be worth it.